DOCS-5584 preview and publish community (#99)
mulesoft/docs-api-community-manager: modules/ROOT/pages/publish-community.adoc

endif::[]
:imagesdir: ../assets/images/

> After you have edited and previewed your community site, you can publish the new version for your community users:

Reviewer: ```suggestion
After you edit and preview your updated community site, you can publish the new version for your users:
```

> . From the API Community Manager (ACM) Control Panel, click *Community Builder*.

Reviewer: ```suggestion
. From the API Community Manager (ACM) control panel, click *Community Builder*.
```

. Click *Publish*.

If your community is not already active, activate it:

> . From the API Community Manager (ACM) Control Panel, click *Community Administration*.

Reviewer: ```suggestion
. From the API Community Manager (ACM) control panel, click *Community Administration*.
```

. Click *Settings*.
. Click *Activate*.

After your community is published and active, view the live site:

> . Open the API Community Manager (ACM) Control Panel.

Reviewer: ```suggestion
. Open the API Community Manager (ACM) control panel.
```
